server/src/game/event/on_grave_added.rs

```diff
@@ -1,14 +1,18 @@
-use crate::game::{grave::Grave, Game};
+use crate::game::{grave::GraveReference, player::PlayerReference, Game};
 
 
 pub struct OnGraveAdded{
-    grave: Grave
+    grave: GraveReference,
 }
 impl OnGraveAdded{
-    pub fn new(grave: Grave) -> Self{
-        Self{ grave }
+    pub fn new(grave: GraveReference) -> Self{
+        Self{grave}
     }
     pub fn invoke(self, game: &mut Game){
+        for player in PlayerReference::all_players(game){
+            player.on_grave_added(game, self.grave.clone())
```

Comment:
Literally turn your linter on bro.
Cloning something that implements copy

Reply:
Fixed